src/commands/iota/create-config.ts

```diff
@@ -0,0 +1,170 @@
+      const walletChoices = wallets.map((wallet: any) => ({
+        name: wallet.name || wallet.id,
+        value: wallet.ari,
+      }))
+
+      const CREATE_NEW_WALLET = 'Create new wallet'
+      walletChoices.push({ name: CREATE_NEW_WALLET, value: CREATE_NEW_WALLET })
+
+      const selectedWallet: string = await select({
+        message: 'Select the wallet used to sign the request token to the Affinidi Vault',
+        choices: walletChoices,
+      })
+
+      if (selectedWallet === CREATE_NEW_WALLET) {
+        const walletDidMethodChoices = Object.values(DidMethods).map((method: string) => ({
+          name: method,
+          value: method,
+          default: DidMethods.KEY,
+        }))
+
+        const didMethod = await select({ message: 'Select DID method of wallet', choices: walletDidMethodChoices })
+
+        const newWalletData = {
+          name: validateInputLength(await input({ message: 'Enter wallet name' }), INPUT_LIMIT),
+          description: validateInputLength(
+            await input({ message: 'Enter wallet description (optional)' }),
+            INPUT_LIMIT,
+          ),
+          didMethod,
+          ...(didMethod === DidMethods.WEB && {
+            didWebUrl: validateInputLength(
+              await input({ message: 'Enter did:web URL (your applications domain)' }),
+              INPUT_LIMIT,
+            ),
+          }),
+        }
+
+        const walletSchema = z.object({
+          name: z.string().min(3).max(INPUT_LIMIT).optional(),
+          description: z.string().max(INPUT_LIMIT).optional(),
+          didMethod: z.nativeEnum(DidMethods).optional(),
+          didWebUrl: z.string().max(INPUT_LIMIT).optional(),
+        })
+        const createWalletInput = walletSchema.parse(newWalletData)
+
+        ux.action.start('Creating wallet')
+        const { wallet } = await cweService.createWallet(createWalletInput as any)
+        ux.action.stop('Created successfully!')
+
+        walletAri = wallet.ari
+      } else {
+        walletAri = selectedWallet
+      }
+    }
+
+    const data: CreateIotaConfigurationInput = {
+      name: flags.name ?? validateInputLength(await input({ message: 'Enter Iota configuration name' }), INPUT_LIMIT),
+      description: flags.description ?? '',
+      walletAri,
+      iotaResponseWebhookURL: flags['response-webhook-url'] ?? '',
+      enableVerification:
+        flags['enable-verification'] ?? (await confirm({ message: 'Enable verification?', default: false })),
+      enableConsentAuditLog:
+        flags['enable-consent-audit-log'] ?? (await confirm({ message: 'Enable consent audit log?', default: false })),
+      tokenMaxAge:
+        flags['token-max-age'] ??
+        (await number({ message: 'Enter token max age, minutes: 1-10', min: 1, max: 10, default: 10 })),
+      clientMetadata: {
+        name: flags['client-name'] ?? '',
+        origin: flags['client-origin'] ?? '',
+        logo: flags['client-logo'] ?? '',
+      },
+    }
```

Comment:
Check for no-input

Reply:
@carlos-affinidi, only `name` and `wallet-ari` are required (and those are checked in no-input), others are optional. 
Can you please clarify what is expected here?